### PR TITLE
Corrected script for some item #21

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**: Implementations + corrections of a bunch of items

### CODE REVIEW

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -101596,6 +102484,40 @@ Body:
...
+    Defense: 7
+    Slots: 1
+    Locations:
+      Garment: true
+    ArmorLevel: 1
+    EquipLevelMin: 30
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      bonus bHit,1+(.@r-5);
```
Comment (Atemo) [inline db/re/item_db_equip.yml:102520]:
```suggestion
      bonus bHit,1+max(1,(.@r-5));
```

File: db/re/item_db_equip.yml [hunk 2]
```diff
@@ -107279,6 +108551,266 @@ Body:
...
+    Defense: 20
+    Slots: 1
+    Locations:
+      Shoes: true
+    ArmorLevel: 1
+    EquipLevelMin: 70
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      bonus2 bSubEle,Ele_Neutral,3+(.@r-6);
```
Comment (Atemo) [inline db/re/item_db_equip.yml:108569]:
```suggestion
      bonus2 bSubEle,Ele_Neutral,3+max(1,(.@r-6));
```

File: db/re/item_db_equip.yml [hunk 3]
```diff
@@ -107625,6 +109157,21 @@ Body:
...
+    Defense: 10
+    Slots: 1
+    Locations:
+      Shoes: true
+    ArmorLevel: 1
+    EquipLevelMin: 30
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      bonus2 bExpAddRace,RC_Angel,5+.@r;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:109174]:
```suggestion
      bonus2 bExpAddRace,RC_Angel,5+getrefine();
```

File: db/re/item_db_equip.yml [hunk 4]
```diff
@@ -108244,6 +109821,21 @@ Body:
...
+    Defense: 10
+    Slots: 1
+    Locations:
+      Shoes: true
+    ArmorLevel: 1
+    EquipLevelMin: 30
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      bonus2 bExpAddRace,RC_Brute,5+.@r;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:109838]:
```suggestion
      bonus2 bExpAddRace,RC_Brute,5+getrefine();
```

File: db/re/item_db_equip.yml [hunk 5]
```diff
@@ -108214,6 +109776,21 @@ Body:
...
+    Defense: 10
+    Slots: 1
+    Locations:
+      Shoes: true
+    ArmorLevel: 1
+    EquipLevelMin: 30
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      bonus2 bExpAddRace,RC_Undead,5+.@r;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:109793]:
```suggestion
      bonus2 bExpAddRace,RC_Undead,5+getrefine();
```

File: db/re/item_db_equip.yml [hunk 6]
```diff
@@ -108184,6 +109731,21 @@ Body:
...
+    Defense: 10
+    Slots: 1
+    Locations:
+      Shoes: true
+    ArmorLevel: 1
+    EquipLevelMin: 30
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      bonus2 bExpAddRace,RC_Insect,5+.@r;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:109748]:
```suggestion
      bonus2 bExpAddRace,RC_Insect,5+getrefine();
```

File: db/re/item_db_equip.yml [hunk 7]
```diff
@@ -153972,6 +156119,84 @@ Body:
...
+      bonus2 bAddSize,Size_All,10;
+      bonus2 bSubRace,RC_Player_Human,5;
+      bonus2 bSubRace,RC_Player_Doram,5;
+      bonus2 bResEff,Eff_Confusion,10000;
+      bonus bVit,5*getskilllv("SU_HISS");
+      if (getskilllv("SU_SPIRITOFLIFE") ==1) {
+         bonus2 bVariableCastrate,"SU_STOOP",-100;
+         bonus2 bVariableCastrate,"SU_LOPE",-100;
+      }
+      /*TODO : Rush Recklessly State (Increase Animal Skill Damage 1% Per 1000 HP (Up to 100%))*/
```
Comment (Atemo) [inline db/re/item_db_equip.yml:156144]: how about
```
autobonus3 "{ .@val = max(100, (Hp/1000)); bonus2 bSkillAtk,"SU_SCAROFTAROU",.@val; etc.. }",1000,15000,"SU_SCAROFTAROU";
```

File: db/re/item_db_etc.yml [hunk 8]
```diff
@@ -41534,6 +41534,20 @@ Body:
...
+    Buy: 20
+    Weight: 10
+    Locations:
+      Garment: true
+    Flags:
+      DropEffect: CLIENT
+    Script: |
+      bonus2 bMagicAtkEle,Ele_Fire,100;
+      bonus2 bSubEle,Ele_Fire,50;
+      /*TODO Imprison State Resist*/
```
Comment (Atemo) [inline db/re/item_db_etc.yml:41550]: I didn't test yet, maybe
```
bonus2 bResEff,SC_WHITEIMPRISON,100;
```
could work?
If it works please export the constant
```
export_constant2("Eff_WhiteImprison",SC_WHITEIMPRISON);
```

File: db/re/item_db_equip.yml (hunk 7 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:156144]: It works

Thanks!

File: db/re/item_db_etc.yml (hunk 8 above)
Comment (idk-whoami) [reply db/re/item_db_etc.yml:41550]: It works

Thanks!

File: db/re/item_db_equip.yml (hunk 6 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:109748]: Thanks!

File: db/re/item_db_equip.yml (hunk 5 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:109793]: Thanks!

File: db/re/item_db_equip.yml (hunk 4 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:109838]: Thanks!

File: db/re/item_db_equip.yml (hunk 3 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:109174]: Thanks!

File: db/re/item_db_equip.yml (hunk 2 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:108569]: Thanks!

File: db/re/item_db_equip.yml (hunk 1 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:102520]: Thanks!

**Comment by Atemo** [comment]
Thanks!

**Pull request merged**